package.json

```diff
@@ -53,7 +55,8 @@
   },
   "private": true,
   "prettier": {
-    "proseWrap": "always"
+    "proseWrap": "always",
+    "singleQuote": true
```

Comment:
I prefer the use of single quotes when possible, hence this new option (FYI).

Reply:
+1